### ISSUE TITLE
Mountname is getting truncated on controller notification sent to subscriber 

### ISSUE THREAD

**Issue by Ziabatcha** [openBackhaul/NotificationProxy#62]
When controller notification is raised, and it is received by NP successfully with full mountname (513559991A)
When the notification is sent to subscriber, mountname is getting truncated as 513559991

Logs:
```
[07:28:58.671] DEBUG (1): received event: {"urn-ietf-params-xml-ns-netconf-notification-1.0:notification":{"urn-opendaylight-params-xml-ns-yang-controller-md-sal-remote:data-changed-notification":{"data-change-event":[{"path":"/urn-TBD-params-xml-ns-yang-network-topology:network-topology/urn-TBD-params-xml-ns-yang-network-topology:topology/urn-TBD-params-xml-ns-yang-network-topology:topology[urn-TBD-params-xml-ns-yang-network-topology:topology-id='topology-netconf']/urn-TBD-params-xml-ns-yang-network-topology:node/urn-TBD-params-xml-ns-yang-network-topology:node[urn-TBD-params-xml-ns-yang-network-topology:node-id='513559991A']/urn-opendaylight-netconf-node-topology:connection-status","data":{"netconf-node-topology:connection-status":"connecting"},"operation":"updated"},{"path":"/urn-TBD-params-xml-ns-yang-network-topology:network-topology/urn-TBD-params-xml-ns-yang-network-topology:topology/urn-TBD-params-xml-ns-yang-network-topology:topology[urn-TBD-params-xml-ns-yang-network-topology:topology-id='topology-netconf']/urn-TBD-params-xml-ns-yang-network-topology:node/urn-TBD-params-xml-ns-yang-network-topology:node[urn-TBD-params-xml-ns-yang-network-topology:node-id='513559991A']/urn-opendaylight-netconf-node-topology:session-id","operation":"deleted"}]},"event-time":"2023-12-14T07:28:58.668165416Z"}}
[07:28:58.678] DEBUG (1): starting notification of 2 subscribers for '/v1/notify-controller-attribute-value-changes', source-stream is OpenDayLight1: http://x.x.x.x:xxxx/rests/notif/data-change-event-subscription/network-topology:network-topology/datastore=OPERATIONAL/scope=SUBTREE/JSON
[07:28:58.678] DEBUG (1): sending subscriber notification to: HTTP://x.x.x.x:xxxx/v1/regard-controller-attribute-value-change with content: {"notification-proxy-1-0:attribute-value-changed-notification":{"counter":1,"timestamp":"2023-12-14T07:28:58.668165416Z","resource":"/core-model-1-4:network-control-domain=live/control-construct=OpenDayLight1/logical-termination-point=513559991/layer-protocol=0/mount-point-1-0:mount-point-pac/mount-point-configuration","attribute-name":"connection-status","new-value":"connecting"}}
```

**Comment by pBittenbringMentopolis** [comment]
This issue is fixed in this tag: v1.0.0_test-4

**Comment by Ziabatcha** [comment]
Verified in  v1.0.0_test-4, closing this issue
